packages/account/src/Sections/Profile/PhoneVerification/phone-verification.scss

```diff
@@ -1,4 +1,19 @@
 .phone-verification {
```

Comment:
let's create a variable 
@mixin flex-column {
    display: flex;
    flex-direction: column;
}

and reuse it in many places:

Reply:
I think if we do that, we need to implement in multiple places as well right 🤔 not just my phone-verification.scss have display: flex; flex-direction: column